crates/driver/src/infra/api/solve/dto/auction.rs

```diff
@@ -10,12 +11,9 @@ use {
 };
 
 impl Auction {
-    pub fn into_domain(self) -> Result<competition::Auction, Error> {
+    pub fn into_domain(self, now: infra::time::Now) -> Result<competition::Auction, Error> {
         Ok(competition::Auction {
-            id: match self.id {
-                Some(id) => Some(FromStr::from_str(&id).map_err(|_| Error::InvalidAuctionId)?),
-                None => None,
-            },
+            id: Some(FromStr::from_str(&self.id).map_err(|_| Error::InvalidAuctionId)?),
```

Comment:
Shouldn't the domain `Auction` object also have a mandatory ID? (Unrelated to this PR - feel free to address this in a follow up).

Reply:
👍